pymathics/natlang/translation.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+
+"""
+Language translation
+
+
+"""
+
+# This is under  Text Normalization in WR. But also in Natural Language Processing,
+# and Linguistic Data. I put here because is the only module tuat uses langid and pycountry
```

Comment:
tuat -> that 

It is perfectly fine to use other sections when that makes the code organization better, e.g. group for a particular Python package like spacy.  

The motivation for the sectioning was to suggest how to organize the code in a way that scales as this grows, and that would make sense for someone familiar with WMA groupings, which after all _are_ someone's attempt to put classify things in a logical, encompassing and  coherent order.